src/info/mod.rs

```diff
@@ -417,6 +417,13 @@ mod tests {
             expected_regex,
             simple_info_str
         );
+
+        let v = serde_json::to_value(info).unwrap();
+        let expected_json: serde_json::Value = serde_json::from_str(
+            &include_str!("../../tests/json/test_verilog_repo.serialize.json"),
+        ).unwrap();
+        assert_eq!(v, expected_json);
```

Comment:
I think it makes sense to actually test against the JSON *string*, not the JSON struct. This would truly make it a snapshot test, and it would be easier to diff the two IMO.

Reply:
Agree, string based testing makes a lot more sense.
See 8ab0bb2